frame/dapps-staking/src/tests.rs

```diff
@@ -6,16 +6,9 @@ use std::str::FromStr;
 
 // TODO: Add checks that verify content of the storage!
 
-fn register(stash: u64, controller: u64, bond_amount: u128, contract: SmartContract<AccountId>) {
-    assert_ok!(DappsStaking::bond(
-        Origin::signed(stash),
-        controller,
-        bond_amount,
-        crate::RewardDestination::Staked
-    ));
-
+fn register(developer: u64, contract: SmartContract<AccountId>) {
```

Comment:
Seems a bit redundant to have a method that's just a wrapper around a different method like this.

Reply:
I agree, it does, but I intend to make use cases with a lot of entries and it will be much more readable. this way Please keep this question in mind until I implement these test use cases